### PR TITLE
DOC: fix typos and add references

### PR DESCRIPTION
**Pull request by sciunto**
I detected several typos in skimage.color.

### CODE REVIEW

File: skimage/color/colorconv.py [hunk 1]
```diff
@@ -1239,6 +1240,12 @@ def rgb2luv(rgb):
     Notes
     -----
     This function uses rgb2xyz and xyz2luv.
+
+    References
+    ----------
+    .. [1] http://www.easyrgb.com/index.php?X=MATH&H=16#text16
```
Comment (soupault) [inline skimage/color/colorconv.py]: Please, add http://www.easyrgb.com/index.php?X=MATH&H=02#text2 also.

Comment (sciunto) [inline skimage/color/colorconv.py]: done.

Comment (soupault) [inline skimage/color/colorconv.py]: Wrong place :).

Comment (sciunto) [inline skimage/color/colorconv.py]: fixed ;)

**Pull request merged**